static/resources/js/samaritan-common.js

```diff
@@ -123,4 +123,18 @@ function getURLParameter(sParam){
 			return sParameterName[1];
 		}
 	}
+}
+
+// function to see if a date is inserted in a valid format
+
+function dateFormatValidator(date){
+    patt = /\d\d.\d\d.\d\d\d\d/i
+    if (date.search(patt) == -1){
+
+        return "False"
+    }
+    else{
+
+        return "True"
+    }
 }
```

Comment:
make sure you end files on new line... this is mostly for version control to be consistent.